utils/src/main/scala/za/co/absa/enceladus/utils/fs/FileSystemVersionUtils.scala

```diff
@@ -127,6 +128,54 @@ class FileSystemVersionUtils(conf: Configuration) {
     }
   }
 
+  /**
+    * Checks if a file is located in HDFS or in the local file system.
```

Comment:
```suggestion
    * Checks if a file is located on the HDFS or the local file system.
```

Reply:
I'd say `on HDFS or the local file system` if you write `on the HDFS` you're giving `HDFS` which is a name (even though its an acronym it is never used as such) a definite article and that doesn't work grammatically.

Reply:
Good to know, thanks!